### ISSUE TITLE
Cancel a series of tasks with SIGINT

### ISSUE THREAD

**Issue by bozdoz** [open-cli-tools/concurrently#452]
Description:

I want to run a series of tasks, that are cancellable.

Expected Behaviour:

If I press CTRL-C, I want the entire script (each of the series of tasks) to be aborted (or not started).

Actual Behaviour:

I need to press CTRL-C multiple times to cancel each process.

Reproduction:

Note the `-m 1` flag to run one process consecutively.

```json
{
"sleep": "concurrently --kill-others-on-fail -m 1 \"sleep 3\" \"echo 'hi'\" \"sleep 3\" \"echo 'there'\" \"sleep 3\" \"echo 'buddy'\""
}
```

```bash
npm run sleep

> concurrently --kill-others-on-fail -m 1 "sleep 3" "echo 'hi'" "sleep 3" "echo 'there'" "sleep 3" "echo 'buddy'"

^C[0] sleep 3 exited with code SIGINT
[1] hi
[1] echo 'hi' exited with code 0
[2] sleep 3 exited with code 0
[3] there
[3] echo 'there' exited with code 0
[4] sleep 3 exited with code 0
[5] buddy
[5] echo 'buddy' exited with code 0
```

Is this a bug, or is there a way to do this that is not obvious?

**Comment by gustavohenke** [comment]
Duplicate of #433

I think it's more of a bug.

**Comment by gustavohenke** [comment]
Sorry, I was a bit hasty marking this as a duplicate of #433.
Signals such as SIGINT that the main process receives work separately from `--kill-others-on-fail`.

**Comment by SrBrahma** [comment]
Up.

"dev": "concurrently -c blue,green -n tsw,next \"bun tsw\" \"next dev\""

I already tried all the combinations of the --kill options but they won't work. I have to press Ctrl+C twice to kill all.

**Comment by gustavohenke** [comment]
🚢 This is now fixed in v9.0.0!
https://github.com/open-cli-tools/concurrently/releases/tag/v9.0.0
